docker/web-and-data/Dockerfile

```diff
@@ -64,7 +65,8 @@ COPY --from=build /cbioportal/requirements.txt /cbioportal/requirements.txt
 
 # install build and runtime dependencies
 # ignore update failures
-RUN  pip3 install -r /cbioportal/requirements.txt
+RUN pip3 install awscli
```

Comment:
I guess this is still a draft PR but just trying to avoid potential issues early: why add `awscli` to the cBioPortal image?

Usually one tries to keep the images as small as possible with one specific purpose. The container running `awscli` should prolly be separate.

To add some context. This image was already too bloated and a good example of what not to do :). The importer should prolly have a separate docker image as well as the validator.

Reply:
@inodb yeah I'm not sure this is going to stay actually, mostly just figuring out how to get the github actions runner to cooperate. I dont think that this `awscli` is necessary but for some reason the `pip3 install -r /cbioportal/requirements` complained about this package missing. I imagine it might be a different issue just being masked as this error.